rts/Game/UI/Groups/GroupHandler.cpp

```diff
@@ -183,7 +184,15 @@ bool CGroupHandler::GroupCommand(int num, const std::string& cmd, bool& error)
 			if (group->units.empty())
 				return false;
 
-			camHandler->CameraTransition(0.5f);
+			float smoothness = 0.5f;
+			// check for optional camera smoothness argument
+			if (extraArgs.size() > 0) {
+				smoothness = StringToFloat(extraArgs[0].c_str(), &error);
```

Comment:
What happens if `smoothness` is zero or negative number?

Reply:
Well, answering my own question it seems like 0 is mostly taken care of in `CameraTransition()`, but negative numbers mostly not. I'd suggest to either clamp that number to be always >= 0, or refuse to do anything if the `smoothness` is negative.